libs/core/langchain_core/tracers/base.py

```diff
@@ -2,38 +2,26 @@
 
 from __future__ import annotations
 
+import asyncio
 import logging
-import sys
-import traceback
 from abc import ABC, abstractmethod
-from datetime import datetime, timezone
 from typing import (
     TYPE_CHECKING,
     Any,
     Dict,
     List,
-    Literal,
     Optional,
     Sequence,
-    Set,
-    Tuple,
     Union,
-    cast,
 )
 from uuid import UUID
 
 from tenacity import RetryCallState
 
-from langchain_core.callbacks.base import BaseCallbackHandler
-from langchain_core.exceptions import TracerException
```

Comment:
Hmm keep this import here with a noqa. Hitting some de-serializatoin issue in the unit tests. I can address this in a separate PR later on?

Reply:
sounds good. Thanks